src/batch-client/index.ts

```diff
@@ -314,6 +315,21 @@ export class ZimbraBatchClient {
 		}).then(Boolean);
 	};
 
+	public documentActionresponse = (type: ActionType, options: ActionOptions) => {
+		const { ids, id, ...rest } = options;
+
+		return this.jsonRequest({
+			name: type,
+			body: {
+				action: {
+					id: id || [ids].join(','),
```

Comment:
do we need two parameters id and ids?

Reply:
not required

Reply:
as this is only used for documentAction response so i have used only id so removed it